src/__stories__/sheet-story.tsx

```diff
@@ -82,13 +82,20 @@ type SheetArgs = {
     title: string;
     subtitle: string;
     description: string;
+    multilineDescription: boolean;
 };
 
 type RadioListSheetArgs = SheetArgs & {
     selectedId: string;
 };
 
-export const RadioList: StoryComponent<RadioListSheetArgs> = ({title, subtitle, description, selectedId}) => {
+export const RadioList: StoryComponent<RadioListSheetArgs> = ({
+    title,
+    subtitle,
+    description,
+    multilineDescription,
```

Comment:
multiParagraph name maybe fit better? 🤔

multilineDescription could be this:
![image](https://github.com/Telefonica/mistica-web/assets/6722153/c786db4a-955e-43af-9553-bafe52d59bc5)